### PR TITLE
docs: update eventarc docs and region tags

### PR DESCRIPTION
**Pull request by grant**
## docs: update eventarc docs and region tags

- Updates Eventarc naming style
- Removes leftover traces of Anthos in the `eventarc/` directory
- Updates region tags
- Updates `gcloud` commands (although these may be removed completely at some point when docs are ready)

### CODE REVIEW

**Comment by grant** [comment]
@averikitsch while you're reviewing this, can I ask a question:

For our Jib instructions (`mvn compile jib:build`) for building our container, should we have this alternative way to build a container the default in our docs? Does Cloud Run do this?

If so, we can work the instructions in our docs under Java:

https://cloud.google.com/eventarc/docs/run/pubsub-quickstart

File: eventarc/README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
-# Cloud Eventarc Java Samples
+# Eventarc Java Samples
```
Comment (averikitsch) [inline eventarc/README.md:1]: nit: remove "Java"

File: eventarc/README.md [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
 
 [![Open in Cloud Shell][shell_img]][shell_link]
 
 [shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
 [shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=blog/README.md
 
-This directory contains samples for Cloud Eventarc.
+This directory contains samples for Eventarc.
```
Comment (averikitsch) [inline eventarc/README.md:8]: nit: this description and the ones in the table aren't very descriptive if you don't know what Eventarc is.

File: eventarc/generic/pom.xml [hunk 3]
```diff
@@ -90,7 +90,7 @@ limitations under the License.
         <version>2.8.0</version>
         <configuration>
           <to>
-            <image>gcr.io/PROJECT_ID/eventarc-generic</image>
+            <image>gcr.io/serverless-com-demo/eventarc-generic</image>
```
Comment (averikitsch) [inline eventarc/generic/pom.xml:93]: nit: remove project ID

**Comment by averikitsch** [comment]
> @averikitsch while you're reviewing this, can I ask a question:
> 
> For our Jib instructions (`mvn compile jib:build`) for building our container, should we have this alternative way to build a container the default in our docs? Does Cloud Run do this?
> 
> If yes, we can work the instructions in our docs under Java:
> 
> https://cloud.google.com/eventarc/docs/run/pubsub-quickstart

Jib is the default we use in the Cloud  Run docs for everything but the quickstart. It is consider best practice over Dockerfiles. The quickstart we wanted consistency in using Dockerfiles and most likely will be updated to use buildpacks anyways. Find an example [here](https://cloud.google.com/run/docs/tutorials/secure-services#shipping-private-service).  Those steps also link to the Jib tutorial for context.

File: eventarc/generic/pom.xml (hunk 3 above)
Comment (grant) [reply eventarc/generic/pom.xml:93]: Done.

**Review by grant — COMMENTED** [review]
Addressed comments. PTAL :)

**Review by averikitsch — APPROVED** [review]

**Pull request merged**